src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfigurationMock.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+
+
+final class CachedAppConfigurationMock {
+	init() {
+		#if !DEBUG
+		preconditionFailure("Do not use this mock in a non-DEBUG environment!")
```

Comment:
I think you could just wrap the whole class in `#if DEBUG`

Reply:
might be. I mostly changed the existing `#if UITESTING` statements to `DEBUG`. I'll review this, thanks!

Reply:
I refactored this and 1-2 other places.